src/components/organisms/UiPhoneNumber/UiPhoneNumber.vue

```diff
@@ -0,0 +1,384 @@
+<template>
+  <section
+    v-if="!isLoading"
+    class="ui-phone-number"
+  >
+    <fieldset class="ui-phone-number__fields">
+      <slot
+        name="legend"
+        v-bind="{ label: translation.phoneNumberLabel }"
+      >
+        <UiText
```

Comment:
Q: why?

Reply:
Why the legend? Why I used the UiText component? Or why what? Could you be more specific? :D